opam-packages-conversion/bin/config.py

```diff
@@ -5,7 +5,7 @@
 os.environ['PATH'] = '%s:%s' % (__dir__, os.environ['PATH'])
 
 GH_ORG_NAME = 'esy-ocaml'
-NPM_SCOPE = 'opam'
+NPM_SCOPE = 'opam-alpha'
 
 OPAM_PACKAGES = os.path.join(os.path.dirname(__file__), '..', 'opam-repository', 'packages')
 
```

Comment:
In the lines below — few modifications to config are needed — `opam_*` to `opam_alpha_*`.